index.html

```diff
@@ -37,6 +37,187 @@ <h1>
 </ul>
 
 </section>
+<!-- work section one -->
+<main class="work-section">
+
+ <section class="profile-one">
+    <div class="profile-image">
+        <img src="images/Snapshoot Portfolio.png">
+    </div>
+    <div class="work-profile">
+        <h2>Tonic</h2>
+
+        <ul class="canopy-section">
+            <li id="cano">CANOPY</li>
+            <li><img src="images/Counter.png"></li>
+            <li>Back End Dev</li>
+            <li><img src="images/Counter.png"></li>
+            <li>2015</li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="work-tech">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javaScript</a></li>
+        </ul>
+<di class="see-project">
+        <a href="#">See Project</a>
+    </di>
```

Comment:
- [ ] Please make sure that you write the correct elements! Here's a typo in your `<div>`, please correct it! 🤓 🔧 

```suggestion
<div class="see-project">
        <a href="#">See Project</a>
    </div>
```